network-api/networkapi/buyersguide/templates/product_page.html

```diff
@@ -1,9 +1,9 @@
 {% extends "./bg_base.html" %}
 
-{% load yes_no env cloudinary %}
+{% load yes_no env cloudinary l10n i18n %}
 
 {% block head_extra %}
-  <meta property="og:title" content="privacy not included - {{ product.name }}" />
+  <meta property="og:title" content="{% blocktrans context 'This can be localized. This is a reference to the “*batteries not included” mention on toys.' %}privacy not included - {{ product.name }}{% endblocktrans %}" />
```

Comment:
```suggestion
  <meta property="og:title" content="{% blocktrans context "This can be localized. This is a reference to the “*batteries not included” mention on toys." %}privacy not included - {{ product.name }}{% endblocktrans %}" />
```